VaticanBAV/et/BAVet131.xml

```diff
@@ -27,52 +27,1023 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    </incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i15.2">
+                                    <locus target="#111v"/>
+                                    <title type="complete" xml:lang="gez">Exposition of the rules of the <foreign xml:lang="gez">wāzemā</foreign>, the praise of the morning, <foreign xml:lang="gez">ʾaryām</foreign> and <foreign xml:lang="gez">mawaddǝs</foreign></title>                                   
+                                    <textLang mainLang="gez"/>
+                                    <incipit xml:lang="gez">
+                                        ንጽሕፍ፡ ነገረ፡ ወግዕ፡ ዘዋዜማ፡ ወዘስብሐተ፡ ነግሕ፡ ዘአርያም፡ ወዘመወድስ፡
+                                    </incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i15.3">
+                                    <locus target="#111v"/>
+                                    <title type="complete" xml:lang="gez">Exposition of the order of the <foreign xml:lang="gez">mawaddǝs</foreign></title>                                   
+                                    <textLang mainLang="gez"/>
+                                    <incipit xml:lang="gez">
+                                        ንጽሕፍ፡ ነገረ፡ ወግዕ፡ ዘሥርዓተ፡ መወድስ።
+                                    </incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i15.4">
+                                    <locus target="#111v"/>
+                                    <title type="complete" xml:lang="gez">Exposition of the order of the praises of the morning for the Saturdays</title>                                   
+                                    <textLang mainLang="gez"/>
+                                    <incipit xml:lang="gez">
+                                        ንጽሕፍ፡ ሥርዓተ፡ ስብሐተ፡ ነግህ፡ ዘሰናብ<surplus>ዕ</surplus>ት፡
+                                    </incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i15.5">
+                                    <locus target="#112r"/>
+                                    <title type="complete" xml:lang="gez">Exposition of the order of the praises of the morning for the feast days</title>                                   
+                                    <textLang mainLang="gez"/>
+                                    <incipit xml:lang="gez">
+                                        ንጽሕፍ፡ ሥርዓተ፡ ስብሐተ፡ ነግህ፡ ዘበዓላት፡
+                                    </incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i15.6">
+                                    <locus target="#112r"/>
+                                    <title type="complete" xml:lang="gez">Exposition of the daily order of the praises of the morning</title>                                   
```

Comment:
Following the pattern from above, maybe "Exposition of the order of the praises of the morning for the ferial days"? It would be more precise